loadouts/functions/fn_addRadio.sqf

```diff
@@ -28,7 +28,7 @@ RETURNS:
 
 params ["_unit", "_kind", "_backpack"];
 
-if (_kind isEqualTo "") exitWith {};
+if (!isPlayer _unit || _kind isEqualTo "") exitWith {};
```

Comment:
I'm not a big fan of this preventing AI from having radios, but can resolve this later if it becomes an issue.